### ISSUE TITLE
How to redirect from / to a specific db/table

### ISSUE THREAD

**Issue by dmick** [simonw/datasette#2027]
Using nginx to redirect public IP to the local uvicorn server as 'normal'.  I can't figure out how to redirect such that '/' results in accessing the one db/table I want to serve; redirecting / to /db/table breaks some of the CSS; fooling with base_url doesn't seem to help.  Can someone explain this, if it's possible?

**Comment by gk7279** [comment]
Can you please help or share your expertise with #2030 ?

**Comment by dmick** [comment]
Was that query to me, @gk7279?

**Comment by gk7279** [comment]
yes @dmick . How did you make your public IP redirect to your uvicorn server?

Instead of nginx, I have apache2 on my GCP VM. Any pointers here are helpful too.

Thanks.

**Comment by dmick** [comment]
I managed to make it work by using nginx's 'exact match' (=) combined with 'prefix match'; that is, match explicitly on `/`, and redirect to `/<db>/<table>`, and then have the normal ProxyPath for the unadorned (prefix-matching) `/`.

```
        location = / {
                return 302 /<db>/<table>;
        }
        location / {
                proxy_pass http://127.0.0.1:8001/;
                proxy_set_header Host $host;
        }
```
